docs/cluster-workflow-templates.md

```diff
@@ -0,0 +1,162 @@
+# Cluster Workflow Templates
```

Comment:
Not sure why this needs to be an entire new doc, especially since it's mostly copy/paste from the other doc. Could you please integrate the new information found here to `docs/workflow-templates.md`?

Reply:
I initially thought of adding a section in `workflow-template.md`. But after I saw `workflow-template.md` which is very focused on explaining `workflow-template` and also it has a lot of information. It would be better to have it as a separate file and simply focus on one thing `cluster-workflow-template`. It has a link to `workflow-template.md` so Users can go to `workflow-template` information.

Reply:
Sure, sounds good 🙂